lib/until.ts

```diff
@@ -0,0 +1,21 @@
+import { action } from "./action.ts";
+import type { Operation } from "./types.ts";
+
+/**
+ * It can be used to treat a promise as an operation. This function
+ * is a replacement to the v3 deprecated `call(promise)` function form.
```

Comment:
I think the callout to the v3 usage makes sense in v3, but in v4, I think we can leave that part out.
```suggestion
 * Treat a promise as an {@link Operation}
```